content/news/2022-10-28-metal-announcement/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Metal Announcement"
+date: 2022-10-28
+categories: ["maplibre-gl-native"]
+authors: [wipfli]
+draft: false
+---
+
+MapLibre GL Native is planning for modularization and migration to Metal, Apple's graphics framework, for rendering on iOS mobile devices.
+
+<p>
+<img src="metal.png" alt="Metal" style="max-width:600px;width:100%">
+</p>
+
+We are happy to announce that Stamen Design, with technical leadership by Wet Dog Weather and funding from AWS, have begun planning for the modularization of MapLibre rendering and a separate proposal for migrating the iOS SDK from OpenGL to Metal.
+
+Stamen provides full-stack cartographic services to clients who rely heavily on projects like MapLibre, which tie all the parts of the modern mapping stack – tiles, styles, and SDKs – together. Steve Gifford from Wet Dog Weather will be the technical lead for this project, and Stephanie May, Damon Burgett, and Camilla Mahon from Stamen will support the effort to get these plans in front of the community.
```

Comment:
I would move info about Stamen to the end. First would be best to describe the *what* is being done, what plans we have, the roadmap, etc.

Reply:
Im also missing some more technical information. Except if this should only be a short announcement.